backend/PythonClient/multirotor/control/simulation_task_manager.py

```diff
@@ -180,6 +180,11 @@ def __populate_drone_and_mission_settings(self, new_setting_dot_json, raw_reques
 
             diff_dict = self.__find_diff(single_drone_setting_copy, self.__DEFAULT_DRONE_FULL_LENGTH)
 
+            # If Barometer settings exist in the input JSON, directly use them.
+            if "Sensors" in single_drone_setting and "Barometer" in single_drone_setting["Sensors"]:
```

Comment:
bug here, when the frontend changes "Pressure factor sigma" to 1, and "Uncorrelated Noise Sigma" to 3, it is interpreted as a string that crashes the UE game.

Reply:
"Barometer": {
                    "SensorType": 1,
                    "Enabled": true,
                    "UpdateFrequency": 51,
                    "PressureFactorSigma": 1,
                    "PressureFactorTau": 3601,
                    "UncorrelatedNoiseSigma": 3,
                    "UpdateLatency": "0",
                    "StartupDelay": "0"
                },

Reply:
note that "UpdateLatency" and "StartupDelay" are strings